src/libmongoc/tests/unified/operation.c

```diff
@@ -2019,20 +2147,29 @@ assert_index_exists (test_t *test,
    mongoc_cursor_t *cursor = NULL;
    const bson_t *index;
    const bson_t *error_doc = NULL;
+   bson_t *opts = NULL;
 
    bp = bson_parser_new ();
+   bson_parser_allow_extra (bp, true);
    bson_parser_utf8 (bp, "collectionName", &collection_name);
    bson_parser_utf8 (bp, "databaseName", &database_name);
    bson_parser_utf8 (bp, "indexName", &index_name);
    if (!bson_parser_parse (bp, op->arguments, error)) {
       goto done;
    }
 
+   opts = bson_copy (bson_parser_get_extra (bp));
+   if (op->session) {
+      if (!mongoc_client_session_append (op->session, opts, error)) {
```

Comment:
Likewise, [`assertIndexExists`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#assertindexexists) does not take a session.

Reply:
Removed.